### ISSUE TITLE
Automatic download of LibriSpeech fails

### ISSUE THREAD

**Issue by hagenw** [audeering/audtorch#30]
### Bug

The automatic download of `LibriSpeech` failed for me during the archive extraction phase due to a wrong path name. Most probably, this also applies to other data sets.

```
0.00B [00:00, ?B/s]Downloading https://openslr.org/resources/12/test-clean.tar.gz to indata/tmp/test-clean.tar.gz
347MB [00:05, 68.1MB/s]                                                                                                                                                                        
Extracting indata/tmp/indata/tmp/test-clean.tar.gz
Traceback (most recent call last):
  File "librispeech.py", line 6, in <module>
    download=True
  File "/home/hagen/.envs/speaker-diarization/lib/python3.5/site-packages/audtorch/datasets/libri_speech.py", line 112, in __init__
    self._download()
  File "/home/hagen/.envs/speaker-diarization/lib/python3.5/site-packages/audtorch/datasets/libri_speech.py", line 149, in _download
    remove_finished=True)
  File "/home/hagen/.envs/speaker-diarization/lib/python3.5/site-packages/audtorch/datasets/utils.py", line 145, in extract_archive
    tar = tarfile.open(filename, 'r:gz')
  File "/home/hagen/.envs/speaker-diarization/lib/python3.5/tarfile.py", line 1577, in open
    return func(name, filemode, fileobj, **kwargs)
  File "/home/hagen/.envs/speaker-diarization/lib/python3.5/tarfile.py", line 1624, in gzopen
    fileobj = gzip.GzipFile(name, mode + "b", compresslevel, fileobj)
  File "/usr/lib/python3.5/gzip.py", line 163, in __init__
    fileobj = self.myfileobj = builtins.open(filename, mode or 'rb')
FileNotFoundError: [Errno 2] No such file or directory: 'indata/tmp/indata/tmp/test-clean.tar.gz'
Exception ignored in: <function WeakValueDictionary.__init__.<locals>.remove at 0x7ff19c2767b8>
Traceback (most recent call last):
  File "/home/hagen/.envs/speaker-diarization/lib/python3.5/weakref.py", line 117, in remove
TypeError: 'NoneType' object is not callable
```

### Steps to reproduce

```python
librispeech = audtorch.datasets.LibriSpeech(
    root='indata',
    sets='test-clean',
    download=True
)
```

### Expected behaviour

Download data, extract it, and return the data set as `librispeech`

### Environment

* OS: Ubuntu
* Python version: 3.5.3
* audtorch version: 0.1.1

**Comment by psibre** [comment]
Is this expected to work with a relative path all the way down? I would expect that at some point, it should call `Path("indata").resolve()` or equivalent.

**Comment by hagenw** [comment]
It should work, when I specify a relative path. But I would expect that every function returns an absolute path. Is this the case for `download_url_list`?

**Comment by hagenw** [comment]
Fixed by #33
